### DISCUSSION TITLE
how to customlize conduit?

### DISCUSSION THREAD

**Discussion by 0x8f701** [ProjectOpenSea/seaport#676] [Q&A]
Now conduit controller use `new Conduit` to create new conduit, which means the conduit code is already fixed in compile time

Anyway to create customlized conduit? We have some NFT which doesn't use `transferFrom` it uses other methods for example `transferWhile...`

**Comment by ryanio** [comment]
For security reasons the conduit cannot be modified, the NFTs have to conform to standard ERC-721 or ERC-1155
